lms/djangoapps/shoppingcart/processors/tests/test_CyberSource2.py

```diff
@@ -294,7 +294,7 @@ def _signed_callback_params(
         # These were captured from the CC test server.
         params = {
             # Parameters that change based on the test
-            "decision": "ACCEPT" if accepted else "REJECT",
+            "decision": decision,
             "req_reference_number": str(order_id),
             "req_amount": order_amount,
             "auth_amount": paid_amount,
```

Comment:
Hmm, still not seeing it.  It looks like `_signed_callback_params()` is still returning a dictionary with an `auth_amount` key.